content/documentation/commands.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "Commands"
+weight = 21
++++
+
```

Comment:
We should provide a small introduction before the different commands, something like:

> Phel includes a series of commands out-of-the-box that helps you to realise certain operations.
> Additionally, you can run `phel list` to see an overview of all commands.